index.js

```diff
@@ -49,51 +52,31 @@ exports.handler = async (event) => {
       const total = bytesDownloaded.reduce((a, b) => a + b, 0)
       const totalSeconds = arr.bytesToSeconds(total)
       const totalPercent = arr.bytesToPercent(total)
-      const hasSeconds = totalSeconds >= minSeconds
-      const hasPercent = totalPercent >= minPercent
-      const overallReason = hasSeconds ? 'seconds' : (hasPercent ? 'percent' : false)
-
-      // start waiting for impressions
-      let waiters = []
-      if (overallReason) {
-        const imp = {...bytesData, bytes: total, seconds: totalSeconds, percent: totalPercent}
-        waiters.push(kinesis.putImpressionLock(redis, imp))
-      } else {
-        waiters.push(false)
+      if (totalSeconds >= minSeconds || totalPercent >= minPercent) {
+        kinesisRecords.push({...bytesData, bytes: total, seconds: totalSeconds, percent: totalPercent})
```

Comment:
I see, not it pushes these on to the `kinesisRecords` array, but hasn't yet sent to kinesis